docs/pages/contribution/contribution.md

```diff
@@ -60,7 +60,17 @@ git push origin main
 
 ### 4. Set up the Development Environment
 
-?? @ProKil
+We recommend using DevContainers to set up your development environment.
+
+#### Using VSCode
```

Comment:
This whole setup is wayyyy too VSCode centric, I don't know if people who don't use vscode would appreciate it

Reply:
I don't have other softwares to test. PyCharm is also supported by Dev Containers. One can contribute their usage to the doc.